src/app/markdown/markdown.component.html

```diff
@@ -0,0 +1,53 @@
+ <ul class="nav nav-tabs" *ngIf="editorActive">
+  <li [class.active]="viewType==='markdown'">
+    <a (click)="onClickMarkdownTab()" class="tab-btn">Markdown</a>
+  </li>
+  <li [class.active]="viewType==='preview'">
+    <a (click)="onClickPreviewTab()" class="tab-btn">
+      {{rendering ? "Loading.." : "Preview"}}
+    </a>
+  </li>
+</ul>
+
+<div class="editor-container">
+  <div class="editor"
+    [class.active]="editorActive"
+    [class.show-less]="!showMore && !editorActive">
+    <span (click)="activeEditor()" class="pficon-edit edit-icon"></span>
+    <p almEditable
```

Comment:
Why the use of editable paragraphs, rather than `<input>` fields? Interacting with inputs are easier than paragraphs.